images/metering-ansible-operator/roles/meteringconfig/tasks/main.yml

```diff
@@ -1,4 +1,13 @@
 ---
-- name: start metering
-  debug:
-    msg: "Hello World (non default)"
+
+- name: Extract MeteringConfig spec
+  copy: content="{{ _metering_openshift_io_meteringconfig.spec }}" dest=/tmp/metering-values.yaml
```

Comment:
might want to register the return code in the case where this fails

Reply:
If it fails, it'll just stop the role here and return the error.

Reply:
I would also move the `content={{...` and `dest=...` to their own lines. i.e.
```
copy:
  content: "{{ _metering_openshift_io_meteringconfig.spec }}"
  dest: /tmp/metering-values.yaml
```

Reply:
But why? Inline is shorter and not very long.